src/dut.rs

```diff
@@ -627,18 +681,25 @@ impl SshInfo {
         port: u16,
         dut_port: u16,
         command: &str,
+        additional_ssh_args: Option<&Vec<&str>>,
     ) -> Result<async_process::Child> {
+        let fixed_args = [
+            "-L",
+            &format!("{}:127.0.0.1:{}", port, dut_port),
+            "-o",
+            "ExitOnForwardFailure yes",
+            "-o",
+            "ServerAliveInterval=5",
+            "-o",
+            "ServerAliveCountMax=1",
+        ];
+        let mut args: Vec<&str> = fixed_args.to_vec();
```

Comment:
dont use an array then to_vec just use the vec![] macro

Reply:
I've tried to use `vec![]` but compiler complained it has short lived object. Maybe I need to hold `&format!("{}:127.0.0.1:{}", port, dut_port)` somewhere.